tests/carddatabase/data/cards.xml

```diff
@@ -31,5 +31,31 @@
                 <pt>4/4</pt>
             </prop>
         </card>
+		<card>
```

Comment:
the indentation in this file is a bit wonky now, we don't use tabs in the rest of the file